standards/naming-domains.md

```diff
@@ -4,25 +4,33 @@ category: Operating services
 # Naming domains
 
 All services with real users and which look like GOV.UK should be on
-`*.service.gov.uk` or `*.service.justice.gov.uk`.
+`*.service.gov.uk` or `*.service.justice.gov.uk`. MOJ infrastructure or
+justice-wide services may be eligible for a `*.justice.gov.uk` domain.
 
-### service.gov.uk
+### `service.gov.uk`
 
 The [service standard](https://www.gov.uk/service-manual/technology/get-a-domain-name)
 already sets out when and how to get a `*.service.gov.uk` subdomain:
 public-facing services which have passed their beta assessment should
 be able to get one.
 
-### service.justice.gov.uk
+### `service.justice.gov.uk`
 
 All other services with real users and which look like GOV.UK should be
 on here, including:
 
 - public-facing services which haven’t yet passed their beta assessment
 (but may be in private beta with a restricted set of real users)
-- internal services which look like GOV.UK (although [they can’t use the
+- internal services, which may look like GOV.UK (although [they can’t use the
 crown, New Transport font etc](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk))
 
+### `justice.gov.uk`
+
+Only services which are core, MOJ-wide infrastructure, like email or
+videoconferencing, or otherwise applicable to users across the entire
+justice system (like the intranet) may use a `*.justice.gov.uk`
```

Comment:
Who makes the decision on this? Can we say who to speak to to get approval for this? And if your service is eligible, is this option always preferable to a `service.justice.gov.uk` subdomain?

Reply:
Good point; I should note somewhere that people should email `domains@`.

To your second point… I think that, yes, eligibility for `justice.gov.uk` implies one should use that instead of `service`? But I'd be open to others' thoughts, because I'm not 100% on that.